cmd/aro/main.go

```diff
@@ -35,8 +33,6 @@ func usage() {
 }
 
 func main() {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
@hawkowl this should be replaced with `rand.New(rand.NewSource(seed))` as per https://tip.golang.org/doc/go1.20#:~:text=the%20standard%20library.-,math/rand,-The%20math/rand ?

Reply:
Why? rand.New returns a new rand source, it doesn't modify the global one. As per the go 1.20 docs, it's automatically seeded.